tests/chart_tests/test_poddisruptionbudget.py

```diff
@@ -11,7 +11,6 @@ def test_houston_pdb_cronjobs(self):
             "charts/astronomer/templates/houston/cronjobs/houston-expire-deployments-cronjob.yaml",
             "charts/astronomer/templates/houston/cronjobs/houston-check-updates-cronjob.yaml",
             "charts/astronomer/templates/houston/cronjobs/houston-cleanup-deployments-cronjob.yaml",
-            "charts/astronomer/templates/houston/cronjobs/houston-check-updates-cronjob.yaml",
```

Comment:
Why was this removed?

Reply:
its seems to be a typo file no 2 and fine no r seems identical